tests/modeladmin/test_checks.py

```diff
@@ -97,6 +98,30 @@ class TestModelAdmin(ModelAdmin):
 
         self.assertIsValid(TestModelAdmin, ValidationTestModel)
 
+    def assertGeneratedDateTimeFieldIsValid(self, *, db_persist):
```

Comment:
Any thoughts about decorating `assertGeneratedDateTimeFieldIsValid` with `isolate_apps` instead of the callers since it's the one that actually creates and registers a model?

```suggestion
    @isolate_apps("modeladmin")
    def assertGeneratedDateTimeFieldIsValid(self, *, db_persist):
```

Reply:
Good idea :+1: